cg/meta/upload/fohm/fohm.py

```diff
@@ -0,0 +1,199 @@
+                self.daily_reports_list
+            ).sort_values(by=["provnummer"])
+        return self._reports_dataframe
+
+    @property
+    def pangolin_dataframe(self) -> pd.DataFrame:
+        """Dataframe with all pangolin rows from multiple cases"""
+        if not isinstance(self._pangolin_dataframe, pd.DataFrame):
+            self._pangolin_dataframe = self.create_joined_dataframe(
+                self.daily_pangolin_list
+            ).sort_values(by=["taxon"])
+        return self._pangolin_dataframe
+
+    @property
+    def aggregation_dataframe(self) -> pd.DataFrame:
+        """Dataframe with all komplettering rows from multiple cases, and additional rows to be used for aggregation"""
+        if not isinstance(self._aggregation_dataframe, pd.DataFrame):
+            self._aggregation_dataframe = self.reports_dataframe.copy()
+        return self._aggregation_dataframe
+
+    @property
+    def daily_reports_list(self) -> List[Path]:
+        if not self._daily_reports_list:
+            for case_id in self._cases_to_aggregate:
+                self._daily_reports_list.append(
+                    self.housekeeper_api.find_file_in_latest_version(
+                        case_id=case_id, tags=["komplettering"]
+                    ).full_path
+                )
+        return self._daily_reports_list
+
+    @property
+    def daily_pangolin_list(self) -> List[Path]:
+        if not self._daily_pangolin_list:
+            for case_id in self._cases_to_aggregate:
+                self._daily_pangolin_list.append(
+                    self.housekeeper_api.find_file_in_latest_version(
+                        case_id=case_id, tags=["pangolin-typing-fohm"]
+                    ).full_path
+                )
+        return self._daily_pangolin_list
+
+    def create_daily_delivery_folders(self) -> None:
+        self.daily_rawdata_path.mkdir(parents=True, exist_ok=True)
+        self.daily_report_path.mkdir(parents=True, exist_ok=True)
+
+    @staticmethod
+    def create_joined_dataframe(file_list: List[Path]) -> pd.DataFrame:
+        """Creates datafame with all csv files used in daily delivery"""
+        dataframe_list = [pd.read_csv(filename, index_col=None, header=0) for filename in file_list]
+        return pd.concat(dataframe_list, axis=0, ignore_index=True)
+
+    def append_metadata_to_aggregation_df(self) -> None:
+        """
+        Add field with internal_id to dataframe
+        """
+
+        self.aggregation_dataframe["internal_id"] = self.aggregation_dataframe["provnummer"].apply(
+            lambda x: self.status_db.samples_by_ids(name=x).first().internal_id
+        )
+        self.aggregation_dataframe["region_lab"] = self.aggregation_dataframe["internal_id"].apply(
+            lambda x: f"{self.lims_api.get_sample_attribute(lims_id=x, key='region_code').split(' ')[0]}"
+            f"_{self.lims_api.get_sample_attribute(lims_id=x, key='lab_code').split(' ')[0]}"
+        )
```

Comment:
I think these would benefit from bing made into utility functions just to describe what is being accomplished. In particular the second one would benefit from a docstring describing what is happening